packages/client/src/exports.ts

```diff
@@ -43,7 +48,6 @@ export { formStorageNodeAssignmentStreamId } from './utils/utils'
 export type { StreamID, StreamPartID, TrackerRegistryRecord } from '@streamr/protocol'
```

Comment:
`TrackerRegistryRecord` is no longer needed in streamr-1.0, can be removed?

Reply:
There seem to be `ProxyDirection` in `@streamr/trackerless-network`, should we export from that instead of `@streamr/protocol`